java/org/contikios/cooja/mspmote/MspMote.java

```diff
@@ -76,7 +76,7 @@
 /**
  * @author Fredrik Osterlind
  */
-public abstract class MspMote extends AbstractEmulatedMote implements Mote, WatchpointMote {
+public abstract class MspMote extends AbstractEmulatedMote<MspMoteType,MspMoteMemory> implements Mote, WatchpointMote {
```

Comment:
Spaces after the commas like in AbstractWakeupMote?

Reply:
Fixed.